testcases/InstallUbuntu.py

```diff
@@ -44,9 +44,6 @@ def runTest(self):
         self.system.goto_state(OpSystemState.PETITBOOT_SHELL)
         self.c = self.system.sys_get_ipmi_console()
         self.system.host_console_unique_prompt()
-        my_ip = self.system.get_my_ip_from_host_perspective()
-        print "# FOUND MY IP: %s" % my_ip
```

Comment:
Why does this need to be removed? This test helps debug some qemu issues at least...